app/windows/Storage/Components/StorageElement.jsx

```diff
@@ -95,6 +95,15 @@ class StorageElement extends React.Component {
         <td>
           {el.stat.CumulativeSize.value} {el.stat.CumulativeSize.unit}
         </td>
+        <td>
+          {
+            el.dag.links
```

Comment:
Do you think we could limit to 2-3 links and the add "..." as last element to show that there are more?